docs/authoring.md

```diff
@@ -162,12 +159,33 @@ Consuming C#/WinRT components from MSIX-packaged applications is supported with
 
 C#/WinRT supports authoring out-of-process components that can be consumed by Windows Runtime compatible languages. Currently, consuming an out-of-process component is supported for managed C# apps with the use of a packaging project. This scenario requires creating a `WinRT.Host.runtimeconfig.json` file, as demonstrated in the [background task component sample](https://github.com/microsoft/CsWinRT/tree/master/src/Samples/BgTaskComponent). Native C++ consumption of out-of-process C# components is not fully supported yet.
 
+### Using Windows App SDK
+
+Authoring WinUI types with C#/WinRT is in development and is supported with some exceptions.
+Component authors can implement WinUI types using the Windows App SDK class library template. 
+Components are consumed by projects made from the various other Windows App SDK templates.
+
+When using a project reference, projects made from the following templates are not supported currently: C++ app with WAP, C++ Unpackaged, and C# Unpackaged. 
+
+Using a package reference to the authored component will work unless authoring a user control, which is not supported yet.
+
+When authoring a custom user control, consuming the component via NuGet package is not supported currently.
+
+If a C++ app is consuming a component that provides a custom user control, an additional entry needs to be made in the `pch.h` header file and in the manifest. 
+
+For the `pch.h` file, add `#include <winrt/MyAuthoredComponent.MyAuthoredComponent_XamlTypeInfo.h>`.
+For the manifest, add an additional `ActivatableClass` entry for `MyAuthoredComponent.MyAuthoredComponent_XamlTypeInfo.XamlMetaDataProvider`.
+
+When using a template for a WAP-style project, the manifest entries should be added to the `Package.appxmanifest`. 
+Otherwise, follow the steps above for adding a `Foo.exe.manifest` -- if you get an error in the linker (LNK) 
+typically this is due to multiple manifests being found, and `app.manifest` can be removed to resolve the issue.
+
+There's no inherent need to create a `runtimeconfig.json`; typically the one automatically created suffices.

```

Comment:
I don't think we need to explicitly say this unless there are specific scenarios to call out where you need to create this file?

Reply:
I agree this might not be necessary, I added it to try to add clarification after the discussion in this post - https://github.com/microsoft/CsWinRT/issues/1088